appengine_config.py

```diff
@@ -91,9 +91,9 @@ def save(self):
     os.path.join(ROOT_PATH, 'third_party', 'beautifulsoup4-4.7.1'),
     os.path.join(ROOT_PATH, 'third_party', 'bleach-3.1.0'),
     os.path.join(ROOT_PATH, 'third_party', 'callbacks-0.3.0'),
-    os.path.join(ROOT_PATH, 'third_party', 'gae-cloud-storage-1.9.15.0'),
-    os.path.join(ROOT_PATH, 'third_party', 'gae-mapreduce-1.9.17.0'),
-    os.path.join(ROOT_PATH, 'third_party', 'gae-pipeline-1.9.17.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'gae-mapreduce-1.9.22.0'),
```

Comment:
Please revert back to the original order of the libraries, that is, `gae-cloud-storage` before `gae-mapreduce`. Thanks!

Reply:
Hi @apb7  Done Thanks!